### DISCUSSION TITLE
ibax members should stop using Twitter for registration. For login, it is necessary to register to the system with password and key words or another login method. people are hesitant to link their Twitter accounts while signing up. you have to develop new input methods.

### DISCUSSION THREAD

**Discussion by Akrepkapo** [IBAX-io/go-ibax#2033] [Ideas] [answered]
ibax members should stop using Twitter for registration. For login, it is necessary to register to the system with password and key words or another login method. people are hesitant to link their Twitter accounts while signing up. you have to develop new input methods.

![Screenshot_2022-08-28-01-00-49-98_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/96292241/187049446-1f700c05-1908-40a0-b57e-f38b0902d056.jpg)

@ibaxio @yolandadadada

**Comment by HalloKyle** [comment, marked as answer]
We use Twitter to avoid some bot accounts, although Twitter's real name system is not good either, as evidenced by Musk's refusal to acquire it, because Twitter also has many bots or small numbers, but at least there is still some threshold, anyway thanks for your advice we are looking for a balance to become IBAX users. But canceling Twitter obviously doesn't help improve anything
